GOVERNANCE.md

```diff
@@ -0,0 +1,154 @@
+
+## Decision making
+
+### Team members 
+
+Team member status may be given to those who have made ongoing contributions for at least 3 months to the Prometheus Project. This is usually in the form of code improvements and/or notable work on documentation, but organizing events or user support could also be taken into account. 
+
+New members may be proposed by any existing member by email to [prometheus-team][team] and voted on by [lazy consensus](#consensus). If no consensus is reached, the new member is not accepted.
+
+Team members are added to the [GitHub organization][gh] and have commit rights to all projects. They should however respect the maintainers of each project.
+
+Team members may retire at any time by emailing [the team][team].
+
+Team members can be removed by [supermajority vote](#supermajority-vote) on [the team mailing list][team]. For this vote, the member in question is not eligible to vote and does not count towards the quorum.
+
+Upon death of a member, their team membership ends automatically.
+
+Unless voted upon otherwise, membership changes take effect immediately.
+
+A public list of team members is kept on the [community page][community] or linked from it.
+
+### Maintainers
+
+Maintainers are team members who lead one or more project(s) and serve as a point of conflict resolution amongst the contributors to this project.
+
+A maintainer or committer may resign by notifying the [team mailing list][team]. A maintainer with no project activity for a year is considered to have resigned. Maintainers who wish to resign are encouraged to propose another team member to take over the project.
+
+An extraordinary change of maintainership of any project may be proposed on the [team mailing list][team] and is voted on by [majority vote](#majority-vote).
+
+A project may have more multiple maintainers, as long as the responsibilities are clearly agreed upon between them. This includes coordinating who handles which issues and pull requests.
+
+Any maintainership change is performed by changing the project's MAINTAINERS.md on the main branch of the project.
+
+### Technical decisions
+
+Technical decisions that only affect a single project are made informally by the maintainer of this project, and [lazy consensus](#consensus) is assumed.
+
+Technical decisions that span multiple parts of the Prometheus Project are discussed and made on the [Prometheus developer mailing list][devs]. Decisions are usually made by [lazy consensus](#consensus). If no consensus can be reached, the matter may be resolved by [majority vote](#majority-vote).
```

Comment:
Usually we end up doing this by lazy consensus within one project (usually Prometheus itself) rather than on the list, and that has generally worked okay. I'd reserve the list for when that doesn't work.

Reply:
How would you formulate that? As a team member, how can I make sure that I am aware of all project-wide decisions?

Reply:
can we settle on "should"? To me, it becomes a cross-project concern at the point where you think we should propagate it for consistency, and is worthy at least an email stating it.

"Follow all discussions in every repository" is not practical advice.

Reply:
For me it's at the initial decision point, as things like that are sticky. Most of these things are pretty minor and not worth increasing mail volume for. The only recent one of note offhand was adding /-/healthy.

> "Follow all discussions in every repository" is not practical advice.

Not for most people, but it's the only real way to spot things like this as they naturally emerge.

Reply:
> Can you propose a formulation for what you have in mind?

The simplest change would be s/are/can be/. I think trying to codify our existing practice precisely would get a bit wordy.

Reply:
adopted that.